src/app/map/crop-production-area-panel/crop-production-area-panel.component.html

```diff
@@ -4,7 +4,7 @@
 
     <section class="map__control__section">
         <h2 class="map__control__title title">
-            <span class="title__text">Crop Production</span>
+            <span class="title__text">{{ 'CROP_PRODUCTION_AREA.TITLE' | translate}}</span>
```

Comment:
Add space after the `translate` pipe like this `{{ 'CROP_PRODUCTION_AREA.TITLE' | translate }} `